CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+## [next-next-version]
+
+### Changed
+
+- Public: Added `autoFocusOnInitialScreenTitle` SDK configuration option for integrators to override SDK auto focusing on the SDK's initial screen title on loading. This default behaviour might not be desireable for some host apps or sites as this could cause the browser to auto scroll to the SDK if there is content that the end user should see first.
```

Comment:
🤓  [discussion]
If it's just the scrolling that is the issue, and focusing the title is always desirable for accessibility, we could simply `preventScroll` and keep the `focus`, i.e.
`containerRef.current?.focus({ preventScroll: true })`

From the linked issue I get the impression that it's not just scrolling though, they also want to keep focus on a different "out-of-SDK" textarea even after the initial SDK screen loads and renders.
If that's the case, we should probably document accordingly, lest we introduce a regression for that use case.

Reply:
That's true. 😅  I've forgotten about the "out-of-SDK" textarea in the discussion when writing this note. Which I think is also the main issue for a particular client that is waiting on this. Will rephrase/edit this CHANGELOG entry.